api/envoy/extensions/http/stateful_session/cookie/v3/cookie.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
+
+package envoy.extensions.http.stateful_session.cookie.v3;
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.http.stateful_session.cookie.v3";
+option java_outer_classname = "CookieProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: Cookie based stateful session extension]
+
+// This extension allows the session state to be tracked via cookies.
+//
+// This extension first encodes the address of the upstream host selected by the load
+// balancer into a `set-cookie` response header with the configured name.
+// This extension will also try to parse the specific upstream host address from
+// the named cookie when new requests come. If the address parsed from the cookie
+// corresponds to a valid upstream host, this upstream host will be selected first.
+// See :ref:`stateful session filter
+// <envoy_v3_api_msg_extensions.filters.http.stateful_session.v3.StatefulSession>`.
```

Comment:
I would love an example (with "real" value) to illustrate this.

Reply:
get it.